microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -3,102 +3,212 @@ description: The latest experimental features in Microsoft Edge DevTools
 ---
 # Experimental features
 
-Microsoft Edge DevTools provide access to experimental features that are still in development.  You can try out these features and [provide feedback](../contact.md) before each feature is released.
+Microsoft Edge DevTools provide access to experimental features that are still in development. This article describes only experimental features found in Microsoft Edge. You can try out these features and [provide feedback](../contact.md) before each feature is released.
 
-All channels of Microsoft Edge have experimental features.  You can get the latest experimental features by using the Microsoft Edge Canary channel.  This article only covers selected experimental features.  For the full list, see the **Settings** > **Experiments** page in DevTools.
+All [channels of Microsoft Edge](/deployedge/microsoft-edge-channels) have experimental features. You can get the latest experimental features by using the [Microsoft Edge Canary channel](https://www.microsoftedgeinsider.com/welcome?channel=canary). To view the full list available in your version of Microsoft Edge, see the **Settings** > **Experiments** page in DevTools.
 
 
 <!-- ====================================================================== -->
-## Experimental features which are turned on by default
-
-As of Microsoft Edge 94, the following experimental features are turned on by default.  This means that these features can be used right away, without you having to change any settings.  You can turn off these default experimental features, if needed.
-
-<!-- listed in this list in order per Settings > Experiments pane -->
-*  Source order viewer
-*  [Emulation: Support dual screen mode](../device-mode/dual-screen-and-foldables.md) is turned on by default starting with Microsoft Edge 90.
-*  Enable webhint
-*  Show issues in Elements
-*  Enable Composited Layers in 3D View
-*  DevTools Tooltips
-*  Visual Studio Code themes for the DevTools
-*  Enable keyboard shortcut editor - [Edit keyboard shortcuts for any action in the DevTools](../customize/shortcuts.md#edit-the-keyboard-shortcut-for-a-devtools-action) is turned on by default starting with Microsoft Edge 89.
-
+## Experimental features in the current release
+
+The following table lists the default status of each experimental feature. The feature name links to more information.
+
+| Default status | Experimental feature |
+| :---: | :--- |
+| Off | [Allow extensions to load custom stylesheets](#allow-extensions-to-load-custom-stylesheets) |
```

Comment:
Note that the UI terminology is "selected" or "cleared", suggesting that the values should read "Selected" and "Cleared", not "On" and "Off" (tho MWSG might say "turn on" in other places) - https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/check-checkbox-check-mark - "Use _select_ and _clear_ with checkboxes, not _turn on_ and _turn off_, mark and unmark, check and uncheck, or select and deselect."

Reply:
deleted table ... but might need to be revisited for the file in general.

Reply:
table deleted